### DISCUSSION TITLE
3D calibration

### DISCUSSION THREAD

**Discussion by WaeberE** [jungmannlab/picasso#418] [Q&A]
Hey, 

I'm using the Picasso compiled version to analyze 3D dSTORM data. As I understood Picasso is using an adapted version described in the paper by Huang et al. (2008) to evaluate the calibration curve. Are the values on the y axis absolute or relative and what are the units? And what values is the Picasso software later using for the z fit on the measured data? I'm wondering if the absolute size of PSF of the calibration beads has an influence on the 3D results. Do you have any experience on that?

Cheers,
Eileen

**Comment by rafalkowalewski1** [comment]
Hi Eileen,

Apologies for a late reply. In the plots you obtain after the 3D calibration, spot width and height are expressed in the camera pixels but the estimatez z positions, their deviations and stage positions are in nm.

The calibration works by fitting a 6th order polynomial where x-axis represents the stage positions (nm) and y-axis represents the spot (PSF) width/height in unita of camera pixels.

I am not sure what you mean by the size of PSF having an impact on the calibration results? Could you please clarify this?

Bests,
Rafal

**Comment by WaeberE** [comment]
Hi Rafal,

Apologies for the late reply! Thank you very much for the answer and clarification. They were helpful.
I noticed that the spot size of the fluorophores differed between the calibration beads that I used and the fluorophores in the sample due to differences in emission intensity.

That’s why I was wondering if Picasso takes that into account when assigning the z position of the fluorophores in the sample. For example, if there’s a z position where the calibration bead has an average spot size of 2 and 1.2 camera pixels in x and y direction, respectively. Will Picasso then assign fluorophore events to this z position only if the PSF has the exact same absolute values in x and y direction (meaning 2 and 1.2 camera pixels) or will it assign events to that position if the ratio between the x and y direction are the same (meaning a x/y ratio of 1.667). In the case of absolute values, there will be a wrong assignment if the sizes of the PSF in the calibration sample and in the measurement sample differ.

I hope this clarifies my question a bit more.

I wish you a Merry Christmas!

Best,
Eileen

**Comment by rafalkowalewski1** [comment]
Hi Eileen,

Picasso takes the absolute values of widths and x and y separately, just as described in section 4 of the sup material in https://doi.org/10.1126/science.1153529. 

Bests,
Rafal
